plugins/com.google.cloud.tools.eclipse.appengine.newproject.maven/src/com/google/cloud/tools/eclipse/appengine/newproject/maven/MavenAppEngineStandardWizardPage.java

```diff
@@ -251,6 +253,38 @@ public boolean validatePage() {
     return true;
   }
 
+  private boolean validateLocation(String location) {
+    if (location.isEmpty()) {
+      setMessage(Messages.getString("PROVIDE_LOCATION"), INFORMATION); //$NON-NLS-1$
+      return false;
+    } else {
+      try {
+        java.nio.file.Path path = Paths.get(location);
+        do {
+          if (Files.exists(path)) {
+            if (!Files.isDirectory(path)) {
+              String message = MessageFormat.format(Messages.getString("FILE_LOCATION"), path.toString()); //$NON-NLS-1$
```

Comment:
No, that;'s quite deliberate. In this example it it path (which can be a parent of location) that is problematic. E.g. in /foo/bar/baz we don't want to say that /foo/bar/baz is a file if in fact it's /foo/bar that's a file and baz does not even exist.